### PR TITLE
Fix \notin

### PR DESCRIPTION
**Pull request by jlchereau**
https://github.com/mathquill/mathquill/issues/739

### CODE REVIEW

**Comment by jlchereau** [comment]
See also #695

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+idea/
```
Comment (laughinghan) [inline .gitignore]: This is specific to your development environment and unrelated to MathQuill, please add this to your global `.gitignore` instead: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Comment by laughinghan** [comment]
If you merge latest `master`, CI should be fixed

**Comment by jlchereau** [comment]
Closed for the reason explained at https://github.com/mathquill/mathquill/issues/739#issuecomment-309214961

**Pull request closed without merging**